### PR TITLE
4 chat room

### PR DESCRIPTION
**Pull request by autto91**
It's a very bare-bones and ugly chat REPL... but you can chat with yourself at least!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+__pycache__
+*.pyc
+tags
```
Comment (armaneous) [inline .gitignore:3]: Add `.vscode`, please (we're also going to need to clean it up from the repo, but I can do that)

File: devrequirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+jedi==0.9.0
+mypy==0.470
+pylint==1.6.5
```
Comment (armaneous) [inline devrequirements.txt:3]: Did you branch off of `aa-dev-setup`? There shouldn't be a `devrequirements.txt` file. Please merge `dev` into your branch and remove excess/duplicate files.

File: main.py [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+      (r'/chatsock$', ChatSocket)
     ],
     template_path = os.path.join(os.path.dirname(__file__), "templates"),
     static_path = os.path.join(os.path.dirname(__file__), "static")
   )
 
 if __name__ == "__main__":
   app = make_app()
-  app.listen(8888)
+  app.listen(8000)
```
Comment (armaneous) [inline main.py]: Any reason for the port number change?

File: main.py [hunk 4]
```diff
@@ -1,23 +1,26 @@
...
   def get(self):
     self.render("index.html")
 
 def make_app():
   return tornado.web.Application(
     [
-      (r"/", MainHandler),
+      (r"/$", MainHandler),
+      (r'/chat$', ChatHandler),
+      (r'/chatsock$', ChatSocket)
```
Comment (armaneous) [inline main.py]: What does `$` mean in this context?

File: devrequirements.txt (hunk 2 above)
Comment (armaneous) [reply devrequirements.txt:3]: Wait, I just saw `requirements.txt` below. Please just remove this file.

**Review by armaneous — CHANGES_REQUESTED** [review]

**Comment by armaneous** [comment]
Sorry! I started the review earlier this week, but forgot I had to submit from the top right after adding all the comments. Bit of a learning curve getting used to this GitHub review format.

File: .gitignore (hunk 1 above)
Comment (autto91) [reply .gitignore:3]: added it to the .gitignore, should be removed now

File: devrequirements.txt (hunk 2 above)
Comment (autto91) [reply devrequirements.txt:3]: removed the devrequirements.txt, I commited it accidentally when switching from my dev-setup branch to dev

**Review by armaneous — APPROVED** [review]

**Review by kammitama5 — APPROVED** [review]
looks good.

**Pull request merged**